kitsune/settings.py

```diff
@@ -1361,4 +1361,12 @@ def filter_exceptions(event, hint):
 WAGTAIL_SITE_NAME = config("WAGTAIL_SITE_NAME", default="Mozilla Support CMS")
 WAGTAILADMIN_BASE_URL = config("WAGTAILADMIN_BASE_URL", default="")
 WAGTAILIMAGES_MAX_UPLOAD_SIZE = IMAGE_MAX_FILESIZE
+WAGTAILADMIN_RICH_TEXT_EDITORS = {
```

Comment:
Why are we overriding this? To add the `code` feature? Where do we need it?

Reply:
See above.